test/src/test/java/com/distkv/dst/test/benchmark/DstBenchmarkTest.java

```diff
@@ -1,7 +1,7 @@
-package org.dst.test.benchmark;
+package com.distkv.dst.test.benchmark;
 
```

Comment:
no blank line

Reply:
I renamed it with IntelliJ IDEA. It generated extra spaces for me. I have manually modified it again.

Reply:
There is o extra blank lines here.

Reply:
There is no extra blank lines here.